cmd2/pyscript_bridge.py

```diff
@@ -7,12 +7,10 @@
 Released under MIT license, see LICENSE file
 """
 
-import argparse
 import sys
-from typing import List, Optional
+from typing import Optional
```

Comment:
There are major changes to the way the **pyscript** command works beyond what we can catch in our unit tests.  Have you performed sufficient manual testing with existing scripts to verify correct behavior?

Reply:
The existing scripts were not affected since they did not use `ArgparseFunctor` or `PyscriptBridge.__getattr__()`. But yes, I did test them.

Reply:
Ok, sounds good enough.